cmd/tailscale/cli/cli_test.go

```diff
@@ -1071,6 +1071,32 @@ func TestUpdatePrefs(t *testing.T) {
 			},
 			env: upCheckEnv{backendState: "Running"},
 		},
+		{
+			name:             "force_reauth_over_ssh_no_risk",
+			flags:            []string{"--force-reauth"},
+			sshOverTailscale: true,
+			curPrefs: &ipn.Prefs{
+				ControlURL:       "https://login.tailscale.com",
+				AllowSingleHosts: true,
+				CorpDNS:          true,
+				NetfilterMode:    preftype.NetfilterOn,
+			},
+			env:          upCheckEnv{backendState: "Running"},
+			wantErrSubtr: "aborted, no changes made",
+		},
+		{
+			name:             "force_reauth_over_ssh",
```

Comment:
Does this cause a 5s wait in the test?

Reply:
No, it doesn't. I don't really understand how this is the case, but I've cribbed this test from the tests like `disable_ssh_over_ssh_no_risk` (line 983 in this file) which is testing similar functionality (that also has a 5s wait when run interactively).

Reply:
Ah, it's because of this:

```
func presentRiskToUser(riskType, riskMessage, acceptedRisks string) error {
	// ...
	if inTest() {
		return errAborted
	}
```